src/main/java/uk/gov/justice/hmpps/casenotes/sync/SyncCaseNoteRequest.kt

```diff
@@ -110,4 +114,8 @@ data class SyncAmendmentRequest(
 
   @Schema(requiredMode = REQUIRED, example = "2024-09-01T10:25:00", description = "Date time of the amendment")
   override val createdDateTime: LocalDateTime,
+
+  @Schema(requiredMode = REQUIRED, description = "Username of the user that created the case note")
+  @NotBlank
+  override val createdByUsername: String,
```

Comment:
I guess they'd set author username and this to the same value extracted from the amendment delimiter